### PR TITLE
[SW-2090] Set log level via rest API

### PR DESCRIPTION
**Pull request by jakubhava**
Created this change to to also reflect the latest changes in log class on master

### CODE REVIEW

File: extensions/src/main/scala/ai/h2o/sparkling/extensions/rest/api/LogLevelHandler.scala [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+/**
+  * This servlet class handles GET and PUT requests for the path /3/LogLevel
+  */
+final class LogLevelHandler extends Handler {
+
+  def setLogLevel(version: Int, request: LogLevelV3): LogLevelV3 = {
+    new MRTask() {
+      override def setupLocal() {
+        if (!Log.LVLS.contains(request.log_level)) {
+          Log.warn(s"[$request.log_level] is not a supported log level.")
```
Comment (mn-mikke) [inline extensions/src/main/scala/ai/h2o/sparkling/extensions/rest/api/LogLevelHandler.scala:35]: nit: Would't be possible to mention in the message that log level is remaining unchanged and mention the original log level?

File: py/src/ai/h2o/sparkling/H2OContext.py [hunk 2]
```diff
@@ -148,6 +148,9 @@ def show(self):
     def getConf(self):
         return self._conf
 
+    def setH2OLogLevel(self, level):
```
Comment (mn-mikke) [inline py/src/ai/h2o/sparkling/H2OContext.py:151]: Shouldn't we also expose a getter?

File: extensions/src/main/scala/ai/h2o/sparkling/extensions/rest/api/LogLevelHandler.scala (hunk 1 above)
Comment (jakubhava) [reply extensions/src/main/scala/ai/h2o/sparkling/extensions/rest/api/LogLevelHandler.scala:35]: 👍

File: py/src/ai/h2o/sparkling/H2OContext.py (hunk 2 above)
Comment (jakubhava) [reply py/src/ai/h2o/sparkling/H2OContext.py:151]: Yup, good idea

**Pull request merged**